lessons/careers_outside_academia/lesson.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Careers Outside Academia
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# Discussion on careers outside academia
+
+ - **Authors**: UofT Coders
+ - **Research field**: Science
+ - **Lesson topic**: Discussion on experiences outside an academic career path
+ - **Lesson content URL**: https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/careers_outside_academia
+
+This is a list of notes and resources that were brought up during our discussion on careers outside academia.
+I did my best to find appropriate links, but some things might not be accurate. Feel free to add to or modify this list!
+
+## Hackathons, coding groups, and places to find them
+* [Civic Tech Toronto](http://civictech.ca/)
+* [Meetup.com](https://www.meetup.com/)
+* [Hacking Health](http://hackinghealth.ca/city/toronto-canada/)
+* Mozilla Global Sprint ([2016 website](https://science.mozilla.org/programs/events/global-sprint-2016)) - annually in June
+* [NeurotechX](http://neurotechx.com/)
+
+## Professional development, courses, summer schools etc.
+* MBA courses at UofT - possibly free for graduate students if taken during registered terms
+* [Graduate Professional Skills Program](https://www.sgs.utoronto.ca/currentstudents/Pages/Professional-Development.aspx) at UofT
+* [RADIANT neurotechnology summer school](https://www.dal.ca/dept/radiant-create.html) - Dalhousie University
+* Sunnybrook lecture series on medical devices - schedule [here](http://sunnybrook.ca/calendar/?c=4&d=11/17/2016)
+* [Mozilla Open Science mentorship program](https://github.com/MozillaFoundation/mentorship-program#mentorship-program)
+* [HackerYou](https://github.com/MozillaFoundation/mentorship-program#mentorship-program)
+* [Bitmaker](https://bitmaker.co/)
+* [Lighthouse Labs](https://lighthouselabs.ca/)
+* [Insight Data Science Fellowship](http://insightdatascience.com/)
+* Second degree programs in computer science - [UBC](https://www.cs.ubc.ca/students/undergrad/programs/second-degree), [McMaster University](http://academiccalendars.romcmaster.ca/preview_program.php?catoid=7&poid=3965&returnto=563), and  [University of Alberta](https://www.ualberta.ca/computing-science/undergraduate-studies/programs-and-admissions) were mentioned
+
+## Ways to "get yourself out there"
+* Make a LinkedIn profile and keep it current
+* Make a blog - post code, write about your research, etc.
+* Contribute to open science on GitHub - [some ideas from Mozilla Science Lab](https://science.mozilla.org/projects)
+* Recruiting firms, such as [Randstad](https://www.randstad.ca/)
+* [devhub.com](http://www.devhub.com/)?
```

Comment:
I think it's devhub.ca? It's Toronto co-working space I think, they also have a meetup page http://www.meetup.com/Devhub/ which is less about selling you on the space